nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -828,6 +828,41 @@ public void errorSerializationTest() {
         .doTest();
   }
 
+  @Test
+  public void errorSerializationEscapeSpecialCharactersTest() {
+    // Input source lines for this test are not correctly formatted intentionally to make sure error
+    // serialization will not be affected by any existing white spaces in the source code.
+    SerializationTestHelper<ErrorDisplay> tester = new SerializationTestHelper<>(root);
+    tester
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:SerializeFixMetadata=true",
+                "-XepOpt:NullAway:FixSerializationConfigPath=" + configPath))
+        .addSourceLines(
+            "com/uber/Test.java",
+            "package com.uber;",
+            "public class Test {",
+            "   Object m = new Object();",
+            "   public void run() {",
+            "     // BUG: Diagnostic contains: passing @Nullable parameter 'm.hashCode()",
+            "     foo(m.hashCode() == 2 || m.toString().equals('\\t') ? \t\n\t\n new Object() : null);",
```

Comment:
Can we make the new lines here explicit in the test case? E.g. instead of passing a test case string with `\n`, just put the new lines inside a new comma-separated string in its own line:

```
" foo(m.hashCode() == 2 || m.toString().equals('\\t') ? \t",
"\t",
" new Object() : null);",
```
?

Reply:
b57eef578749b860ee06e1f79e59319a8c3fcf6a